Documentation/gettingstarted/bandwidth-manager.rst

```diff
@@ -0,0 +1,120 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bandwidth-manager:
+
+*****************
+Bandwidth Manager
+*****************
+
+This guide explains how to configure Cilium's bandwidth manager to in order to
+optimize TCP and UDP workloads and efficiently rate limit individual Pods if
+needed through the help of EDT and BPF.
+
+The bandwidth manager is natively integrated into Cilium and does not make use
+of the `bandwidth CNI
+<https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-traffic-shaping>`_
+plugin. Due to scalability concerns, it is not recommended to use the bandwidth
+CNI plugin.
+
+Cilium's bandwidth manager supports the ``kubernetes.io/egress-bandwidth`` Pod
+annotation which is enforced on egress at the native host networking devices.
```

Comment:
veth devices are `noqueue` devices by default, it would be broken to somehow enforce queueing on tc ingress in the host. It's possible via ifb, but completely broken architecture. The only point that makes sense is the tc egress on the phys dev.

Reply:
Ah, I didn't know that, but yeah, it makes sense :slightly_smiling_face: